test-projects/01-basic-app/tests/acceptance/start-mirage-test.js

```diff
@@ -52,6 +53,32 @@ module('Acceptance | Starting mirage', function(hooks) {
       assert.dom('[data-test-id="user"]').exists();
     });
 
+    module('nested mirage modules', function() {
+      test('it works', async function(assert) {
+        const server = startMirage(this.owner);
+        const model = server.create('nested/thing');
+
+        assert.ok(model instanceof NestedThingModel, 'models');
+        assert.equal(model.id, 'nested identity manager works!', 'identity managers');
+        assert.equal(model.name, 'nested factory works!', 'factories');
+
+        const { attributes } = server.serializerOrRegistry.serialize(model).data;
+        assert.ok('nested_thing_name' in attributes, 'serializer');
+      });
+
+      // factories and fixtures have to be tested separately
```

Comment:
well, we could test it in a single test, but:
 - a nested identity manager does only support a single ID
 - it just looks cleaner to separate